etc/deploy/compose/compose.yaml

```diff
@@ -2,7 +2,7 @@ services:
   postgres:
     image: docker.io/library/postgres:16
     ports:
-      - "5432:5432"
+      - "5433:5432"
```

Comment:
:question:

Reply:
Fixed. Sorry about that, local change as I run other (guac) db at 5432